Editor/Utilities/SerializableInterfaceAdvancedDropdown.cs

```diff
@@ -16,6 +16,7 @@ internal sealed class SerializableInterfaceAdvancedDropdown : AdvancedDropdown
         private readonly Type interfaceType;
         private readonly MethodInfo sortChildrenMethod;
         private readonly bool canSort;
+        private readonly bool rawReferencesOnly;
```

Comment:
I think it would be better to rename this to `classesOnly` to make it match with the `ClassesOnlyAttribute`!

Reply:
Sure thing - I initially tried to keep the internal naming style, but I agree this is more consistent!